### ISSUE TITLE
uncomfortable horizontal scroll appeared by long input gesture label.

### ISSUE THREAD

**Issue by ungjinPark** [nvaccess/nvda#13614]
### Is your feature request related to a problem? Please describe.

I'm low vision but I use a computer looking at the screen yet because I can use it faster than using screen reader. I've a complaint about the tree list in Input Gesture Dialog.

Currently, a label of each tree items is too long in Gesture settings. because most items were written by description style.
If sentence is too long, it makes me users must use horizontal scroll. Advanced users perform horizontal scroll to easy but, Users who unfamiliar with Windows UI and Input devices are maybe uncomfortable.

![screen that horizontal scroll had appeared when tree-item width is overflowed](https://user-images.githubusercontent.com/58059043/163712548-c3868d09-4d3f-4b99-b9d5-42195d1b8825.png)

it's not that there's no way for avoiding horizontal scroll currently. it will be gone if users resize the window width more widely, of course.

But, in my case, I can view a screen by a limitted sight because my right eye is a blind and left is also high myopia. Wide large screen, window-width are rather poisonous for someone like me. I hope the layout of this dialog will be improved.

### Describe the solution you'd like

Each gesture labels in Input Gesture settings dialog are too long. needs more short simplified label. It's not request for removing old strings. I request addition strings for using on gesture tree-items.

### Describe alternatives you've considered

First, Add an alternative short label for each items, for Example,  replace "Moves the review cursor to the first character of the line where it is situated in the current navigator object and speaks it" to "review a first character of the line". And then recycle current string as description.

Second, make layout more legible than now. I hope layout that has two-columns configured treeview list, and description panel. Left side is tree list, right side is panel for displaying gesture description. In the description panel, display a old long description for focused tree item.

### Additional context
None

**Comment by XLTechie** [comment]
CC @Qchristensen

**Comment by feerrenrut** [comment]
While it may not be a solution for you @ungjinPark, it may be helpful to know that the dialog can be resized / maximized.

I agree there are improvements that could be made here. As separate PR's we will accept:
1. Making the descriptions wrap within the dialog.
2. Giving a short name to each of the "scripts". These need to be translated strings, the script name will be a good starting point.
3. Once all the scripts have a short name, update the GUI to show the name primarily. The description should still be displayed somewhere. The developer who works on this will need to consider the UI, present some options, and get feedback from core devs / community about the preferred approach.

**Comment by Adriani90** [comment]
cc: @seanbudd, @hwf1324

**Comment by seanbudd** [comment]
@Adriani90 - this issue is currently triaged, could you please clarify the intention of your cc?

**Comment by Adriani90** [comment]
The last triage was more than 3 years ago, I thought maybe you have an internal triaging review process given the fact that roadmaps generally change over time. So Do I understand correctly that once NV Access triaged an issue, this will never be reviewed or touched again?

**Comment by seanbudd** [comment]
@Adriani90 - could you please clarify what you are expecting us to do in this process? We don't update the state of an issue (e.g. priority) unless reasoning is provided. The purpose of our triage process is to ensure tickets are not blocked on NV Access, so that the community is able to work on them. We only attempt to take on p1s and some limited p2s through the triage process. The rest is generally up to the community or scheduled projects. The role of NV Access as maintainers is generally to ensure the community is empowered as much as possible. We don't have the capacity to solve most issues (as there are thousands), so our focus is more on building the collective capacity of the community and vital infrastructure work.

**Comment by CyrilleB79** [comment]
> We don't update the state of an issue (e.g. priority) unless reasoning is provided. The purpose of our triage process is to ensure tickets are not blocked on NV Access, so that the community is able to work on them. We only attempt to take on p1s and some limited p2s through the triage process. The rest is generally up to the community or scheduled projects. The role of NV Access as maintainers is generally to ensure the community is empowered as much as possible. We don't have the capacity to solve most issues (as there are thousands), so our focus is more on building the collective capacity of the community and vital infrastructure work.

@seanbudd this is very clear. It answers to the following questions:
* What is the goal of the triage process?
* What is the role of NV Access in NVDA's development? What do they generally do or doesn't? Which task/action do they priorize?

Does all this information appear in the repo's documentation (e.g. in the triage doc)? If not, it would be worth adding it somewhere.

Also cc @gerald-hartig

And sorry for the off-topic.

**Comment by seanbudd** [comment]
I think the goals of the triage process are already well covered in [the process doc](https://github.com/nvaccess/nvda/blob/master/projectDocs/issues/triage.md). If you have any suggestions to update this please open a new issue or PR.

In terms of the role of NV Access, the answer I provided was to give a broad picture. This is not a hard rule, or fixed policy. Development priorities of NV Access are relatively dynamic depending on the context of pressing issues and work on our plate

**Comment by Adriani90** [comment]
@seanbudd thanks for your answer.

> We don't have the capacity to solve most issues (as there are thousands), so our focus is more on building the collective capacity of the community and vital infrastructure work.

I fully understand that, I was just thinking about the magnification integration in NVDA and at some point you guys said that there is a prototype in work at NV Access, so I thought this particular issue might fit on to the roadmap for that. That's why I assumed you might want to recheck the triaging.
